src/interfaces/events/types.ts

```diff
@@ -0,0 +1,18 @@
+import type { BaseMessage } from '../../core/types.js';
+import type { BaseMessageReply } from '../../core/message-reply.js';
+import type { Event } from '../../event-log/event-log.js';
+import type { DwnInterfaceName, DwnMethodName } from '../../core/message.js';
+
+export type EventsGetDescriptor = {
```

Comment:
should this `& Descriptor`?

or are you intentionally trying to avoid `dataCid` and `dataSize`?  if so, could we use `Pick` so at least there's some continuity?

Reply:
intentionally avoiding `dataCid` and `dataSize` because they'll never exist in an `EventsGetDescriptor`